ivadomed/random.py

```diff
@@ -0,0 +1,16 @@
+import torch
+import random
+import numpy as np
+
+def set_seed(seed=6, rank=0):
+    """
+    This function controls sources of randomness to aid reproducibility.
+    Args:
+        seed: int to initialize random number generator (RNG)
+        rank: int specifying the GPU id for training
+    Returns:
+        seeded RNG in the specified libraries/packages.
```

Comment:
Can probably remove this line, as there is no "return" _per se_.  Sorry for being pedantic.